evo_researcher/benchmark/benchmark.py

```diff
@@ -130,8 +138,8 @@ def get_prediction_result(market: Market):
                         prediction=prediction,
                         market_question=market_question,
                     )
-                if self.cache_path:
-                    self.predictions.save(self.cache_path)
+                    if self.cache_path:
+                        self.predictions.save(self.cache_path)
```

Comment:
Save after every prediction, handy in case of problems if you are running some longer benchmark.